src/app/components/control-bar/control-bar.component.ts

```diff
@@ -151,13 +167,15 @@ export class ControlBarComponent implements OnInit {
 
       if (this.projects.length) {
         let lastProj;
-        try {
-          // restores view to the last visited project from local storage
-          lastProj = JSON.parse(window.localStorage.getItem('lastProj'));
-          console.log(lastProj);
-        } catch (error) {
-          lastProj = this.projectsService.setActiveProject(this.projects[0]);
-        }
+        // try {
+        //   //restores view to the last visited project from local storage
+        //   lastProj = JSON.parse(window.localStorage.getItem('lastProj'));
+        //   // console.log(lastProj);
+        // } catch (error) {
+        //   lastProj = this.projectsService.setActiveProject(this.projects[0]);
+        // }
+
+        lastProj = this.projectsService.setActiveProject(this.projects[0]);
```

Comment:
```suggestion
        // Opening first project instead of the most recent project as we don't handle deleted projects
        // See https://jira.tacc.utexas.edu/browse/DES-2336
        lastProj = this.projectsService.setActiveProject(this.projects[0]);
```